contracts/multisig-prover/src/migrate.rs

```diff
@@ -0,0 +1,68 @@
+use axelar_wasm_std::hash::Hash;
+use cosmwasm_std::{DepsMut, Response};
+
+use crate::state::{Config, CONFIG};
+
+pub fn set_domain_separator(
+    deps: DepsMut,
+    domain_separator: Hash,
+) -> Result<Response, axelar_wasm_std::ContractError> {
+    let old_config = CONFIG.load(deps.storage)?;
+    let new_config = Config {
+        domain_separator,
+        ..old_config
+    };
+    CONFIG.save(deps.storage, &new_config)?;
+
+    Ok(Response::default())
```

Comment:
Do our existing multisig prover instances have a domain separator? If not, I think this will fail, because it won't be able to deserialize the config.

Reply:
ya, need to provide the old config struct to deserialize

Reply:
fixed, I manually created the old struct and migrated instead of something like default deserialization of the missing field. I feel the default deserialization could lead to unintended side effects